app/code/Magento/Bundle/view/frontend/templates/sales/order/items/renderer.phtml

```diff
@@ -6,7 +6,9 @@
 // phpcs:disable Magento2.Templates.ThisInTemplate.FoundThis
 /** @var $block \Magento\Bundle\Block\Sales\Order\Items\Renderer */
 $parentItem = $block->getItem();
-$items = array_merge([$parentItem], $parentItem->getChildrenItems());
+$parentItemId = $parentItem->getId();
+$childrenItems = $block->getOrder()->getItemById($parentItemId)->getChildrenItems();
+$items = array_merge([$parentItem], $childrenItems);
```

Comment:
Since `$parentItemId` and `$childrenItems` have no usage on the template, I would recommend moving this logic to the block instead.

Reply:
Have been moved to block.